docs/howto/features/buckets.md

```diff
@@ -106,6 +106,35 @@ user_buckets = {
 }
 ```
 
+## Enabling Requester Pays flag on buckets
+
+Some hubs want to expose a particular bucket to the broad internet
+but not be billed for the charges associated with making and executing the
+requests on this bucket.
+
+By enabling the [Requester Pays flag](https://cloud.google.com/storage/docs/using-requester-pays#using),
+the requesters are required to include a billing project in their requests,
+which will mean that the billing will happen on the requester's project.
+
+Enabling Requester Pays is useful, for example, if the communities have a lot
+of data that they want to make available to others, but don't want to be
+charged for their access to that data.
```

Comment:
... but this section sounds like the host hub is providing access to buckets **inside** their hub. Which is it? Can it be both? In what scenario is the bucket **inside** and what scenario is the bucket **outside**?

Reply:
Yes, it can be both. `Requestor Pays` is a feature that a bucket can have.

- If buckets **outside** the project have this flag, then we need to:
     - set `hub_cloud_permissions.allow_access_to_requestor_pays_buckets` in the terraform config of a cluster (this config is simply called `requestor_pays` without this PR, which was the source of confusion for me).
     - This is to allow them to be charged on their project for access of such **outside** buckets
- But buckets that we set for communities, **inside** their projects can also have this feature enabled on them, which means that other people outside will be charged for their usage.
    - This can be enabled by setting `user_buckets.<bucket_name>.requester_pays` to true in the terraform config of the cluster.
    - I don't think we have any buckets like this yet, but was recently requested by leap in https://2i2c.freshdesk.com/a/tickets/1316

Reply:
Does the above has more sense @sgibson91? I will update the language in the docs as well to make the split and differences more clear.

Reply:
Yeah, thank you - though I'm wondering if it should be `hub_cloud_permissions.allow_access_to_external_requestor_pays_buckets` to be crystal clear? Though I realise it's getting kinda long.

Reply:
I just want to say that https://github.com/2i2c-org/infrastructure/pull/3431/files#r1495651158 is an accurate description of the situation! Only difference is that right now,

> This can be enabled by setting user_buckets.<bucket_name>.requester_pays to true in the terraform config of the cluster.

is not true because we don't have that implemented.